packages/clerk-js/src/ui/components/OrganizationList/shared.tsx

```diff
@@ -8,15 +8,22 @@ import { SwitchArrowRight } from '../../icons';
 import type { ThemableCssProp } from '../../styledSystem';
 import { common } from '../../styledSystem';
 
-export const PreviewListItems = (props: PropsWithChildren) => {
+export const PreviewListItems = (
+  props: PropsWithChildren<{
+    sx?: ThemableCssProp;
```

Comment:
You are not using this anywhere 🤔

Reply:
Yeah this is a leftover from an attempt not needed, updating the code!